### ISSUE TITLE
prefer-times issue

### ISSUE THREAD

**Issue by Delagen** [wix/eslint-plugin-lodash#73]
```
var forms = _.map(arr, function(inv) {
     var f = _.cloneDeep(form);
     f.fields = _.map(f.fields, function(field) {
      field = formFunctions.setValue(field, inv.fields, {uniteAmount: false});
      if (field.id === "PenaltyId") {
       f.formSelector.value = field.value;
      }
      return field;
     });
     return f;
    });
```
does not see that inv used inside iteratee
I know that code is ugly, but...

**Comment by ganimomer** [comment]
This was an actual bug - `prefer times` didn't account for usages that might be in a deeper `_.map`.
Fixed and released in v1.8.1.

**Comment by Delagen** [comment]
Thanks
